src/diffusers/loaders.py

```diff
@@ -1279,7 +1324,8 @@ def save_function(weights, filename):
         save_function(state_dict, os.path.join(save_directory, weight_name))
         logger.info(f"Model weights saved in {os.path.join(save_directory, weight_name)}")
 
-    def _convert_kohya_lora_to_diffusers(self, state_dict):
+    @classmethod
+    def _convert_kohya_lora_to_diffusers(cls, state_dict):
```

Comment:
Moved to a class method so can be called from other `load_lora_weights_` as a classmethod. Does not touch any instance state